examples/sdk/train_api.ipynb

```diff
@@ -12,12 +12,13 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.        dataset_provider_parameters=S3DatasetParams(
</pre><p> </p><p>What is this endpoint and credentials? Are you ok to expose these information?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/training-operator/pull/2006/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>